new-pipeline/style/dashboards.css

```diff
@@ -126,10 +126,26 @@
 .mg-marker-text { font-size: 12px !important; }
 .mg-markers line { stroke-width: 2px !important; }
 
-#dist-caption {
-  font-size: 1.5em;
+.caption-div {
+  display: flex;
+  justify-content: center;
+  align-items: center;
   text-align: center;
-  margin: 0.1em;
+  margin-bottom: 3em;
+}
+
+.overflow {
+	white-space: nowrap;
```

Comment:
The whitespace appears to still be off. It looks like it's a tab character instead of spaces?